dask/bag/core.py

```diff
@@ -183,6 +183,29 @@ def finalize_item(results):
     return results[0]
 
 
+def unpack_kwargs(kwargs):
+    """ Extracts dask values from kwargs
+
+    Currently only dask.bag.Item and python literal values are supported.
+
+    Returns a merged dask graph and a list of [key, val] pairs suitable for
+    eventually constructing a dict.
+    """
+    dsk = {}
+    kw_pairs = []
+    for key, val in iteritems(kwargs):
+        if isinstance(val, Item):
+            dsk = merge(dsk, val.dask)
```

Comment:
I was groaning about impure use earlier, which is possibly why @jcorbin is doing it this way.  I agree with @jcrist though that, because we're only mutating a locally created `dict`, rather than something that was passed in, `update` is probably slightly preferred, especially as graphs can become large.